index.bs

```diff
@@ -249,9 +259,39 @@ If the algorithm using these shorthands is running <a>in parallel</a>, the short
 
 <h3 id="aggregating-promises">Aggregating Promises</h3>
 
-The result of <dfn id="waiting-for-all" export>waiting for all</dfn> of a collection of promises is a promise created by calling <code>Promise.all(promiseArray)</code>, where <code>promiseArray</code> is that collection in array form and we use the initial value of <a href="https://tc39.github.io/ecma262/#sec-promise.all"><code>Promise.all</code></a>.
-
-This phrase is useful when you wish to perform multiple asynchronous operations in parallel that return promises, and then react to them all together. If all of the given promises fulfill, then the resulting promise will be fulfilled with an array corresponding to the fulfillment values. If any of them reject, then the resulting promise will be rejected with the first rejection reason to occur.
+To <dfn id="waiting-for-all" export lt="wait for all|waiting for all">wait for all</dfn> of a <a>list</a> of promises |promises|, with success steps |successSteps| that take a <a>list</a> of JavaScript values and failure steps |failureSteps| that take a rejection reason JavaScript value, perform the following steps:
+
+1. Let |rejectionHandler| be a built-in function that takes an argument |arg| and performs the following steps:
+    1. Perform |failureSteps| given |arg|.
```

Comment:
If multiple promises reject, then failureSteps will be performed multiple times. Is this intentional?

Reply:
No, good catch. This was a no-op in Promise.all, because the failure steps there are just to reject a promise, and rejecting multiple times does nothing. Here it is bad. I will fix.